enterprise/server/util/heartbeat/heartbeat.go

```diff
@@ -11,47 +11,71 @@ import (
 
 const (
 	// This node will send heartbeats every this often.
-	heartbeatPeriod = 1 * time.Second
+	defaultHeartbeatPeriod = 1 * time.Second
 
 	// After this timeout, a node will be removed from the set of active
 	// nodes.
-	heartbeatTimeout = 3 * heartbeatPeriod
+	defaultHeartbeatTimeout = 3 * defaultHeartbeatPeriod
 
 	// How often this node will check if heartbeats are still valid.
-	heartbeatCheckPeriod = 100 * time.Millisecond
+	defaultHeartbeatCheckPeriod = 100 * time.Millisecond
 )
```

Comment:
nit: move / copy these comments to the HeartbeatChannel fields below?

Reply:
Done.